posthog/models/filters/test/test_filter.py

```diff
@@ -337,6 +324,33 @@ def test_json_object(self):
             self.assertEqual(events[0]["id"], event1.pk)
             self.assertEqual(len(events), 1)
 
+        def test_element_selectors(self):
```

Comment:
Maybe throw in more events for noise. the test could hypothetically pass from just querying all since there's only 1 event

Reply:
There's 2 events

Reply:
yep sorry wasn't reading properly